client/client-multi/client-c/src/kaa_platform_protocol.c

```diff
@@ -14,9 +14,15 @@
  * limitations under the License.
  */
 
-#include "kaa_platform_protocol.h"
 
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
```

Comment:
Are you sure you need all these headers here? Please cleanup to bare minimum (not only here).